cpu/esp32/vendor/esp-idf/soc/rtc_time.c

```diff
@@ -65,7 +64,7 @@ static uint32_t rtc_clk_cal_internal(rtc_cal_sel_t cal_clk, uint32_t slowclk_cyc
     } else {
         expected_freq = 150000; /* 150k internal oscillator */
     }
-    uint32_t us_time_estimate = (uint32_t) (((uint64_t) slowclk_cycles) * MHZ / expected_freq);
+    uint32_t us_time_estimate = (uint32_t) (MHZ(slowclk_cycles) / expected_freq);
```

Comment:
I am not sure if `uint32_t` is sufficient when `slowclk_cycles` is multiplied by one million and then divided by `expected_freq`. I don't know what the maximum value of `slowclk_cycles` is.